scripts/kind-e2e/e2e.sh

```diff
@@ -87,10 +87,46 @@ function kind_import_images() {
         kind --name cluster${i} load docker-image submariner:local
         kind --name cluster${i} load docker-image submariner-route-agent:local
     done
+
+    if [ ! -z "${global_CIDRs[cluster1]}" ]
+    then
+        docker tag quay.io/submariner/submariner-globalnet:$VERSION submariner-globalnet:local
+        for i in 1 2 3; do
+            echo "Loading globalnet image into cluster${i}..."
+            kind --name cluster${i} load docker-image submariner-globalnet:local
+        done
+    fi
+}
+
+function get_globalip() {
+    svcname=$1
+    gip=
+    attempt_counter=0
+    max_attempts=5
+    # It takes a while for globalIp annotation to show up on a service
+    until [[ $gip ]]; do
+        sleep 1
+        gip=$(kubectl get svc nginx-demo -o jsonpath='{.metadata.annotations.submariner\.io/globalIp}')
+        if [[ ${attempt_counter} -eq ${max_attempts} ]];then
+          echo "Max attempts reached, failed to get globalIp!"
+          exit 1
+        fi
+        attempt_counter=$(($attempt_counter+1))
+    done
+    echo $gip
```

Comment:
Let me suggest an alternative construct which will be clearer:

```suggestion
# It takes a while for globalIp annotation to show up on a service
function get_globalip() {
    svcname=$1
    gip=
    for i in {1..5}; do
        gip=$(kubectl get svc $svcname -o jsonpath='{.metadata.annotations.submariner\.io/globalIp}')
        if [ "$gip" != "" ]; then
            echo $gip
            return
        fi
    done
    
    echo "Max attempts reached, failed to get globalIp!"
    exit 1
```

P.S. you can skip the echo & exit in the end since you already check them externally.

Reply:
I was doing exactly the same thing (i.e., moved to for loop instead of until). I feel that let's keep the echo/exit atleast for sometime until we fully test globalnet.